runtime/westend/src/lib.rs

```diff
@@ -764,20 +735,11 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllModules,
-	UpgradeSessionKeys,
+	CustomOnRuntimeUpgrade,
```

Comment:
```suggestion
	()
```

You can just remove `CustomOnRuntimeUpgrade` and pass either `()` here or just nothing will also work.